packages/core/admin/admin/src/components/MainNav/NavUser.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { Initials, Flex, ButtonProps, VisuallyHidden } from '@strapi/design-system';
+
+export interface NavUserProps extends ButtonProps {
+  id: string;
+  initials: string;
+  children?: React.ReactNode;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+export const NavUser = React.forwardRef<HTMLButtonElement, NavUserProps>(
+  ({ children, initials, ...props }, ref) => {
+    return (
+      <Flex paddingTop={3} paddingBottom={3} ref={ref} justifyContent="center" {...props}>
+        <Flex as="button" justifyContent="center">
+          <Initials>{initials}</Initials>
+          <VisuallyHidden>
+            <span>{children}</span>
```

Comment:
I'm not sure there's a benefit to wrapping `children` in a `span` within `VisuallyHidden` is there?

Reply:
Do you mean completely removing the Visually hidden content?

Reply:
no i mean i would just do:

```
<VisuallyHidden>{children}</VisuallyHidden>
```